src/base/error_code.h

```diff
@@ -1725,8 +1725,9 @@
 #define ER_SYNONYM_INVALID_VALUE                    -1350
 
 #define ER_SM_INVALID_FUNCTION_EXPRESSION_LENGTH    -1351
+#define ER_USING_INDEX_HINT_IGNORE                  -1352
```

Comment:
It's trivial.

Index hints used in the query syntax are "use index" and "using index".
Wouldn't it be confusing because the name of "ER_USING_INDEX_HINT_IGNORE" only contains "using index"?

Reply:
The error message of this error code is too general.

Reply:
I corrected the error code and message with the help of hgryoo.